pkg/plugin/v3/scaffolds/internal/templates/makefile.go

```diff
@@ -72,11 +70,24 @@ endif
 all: manager
 
 # Run tests
-ENVTEST_ASSETS_DIR=$(shell pwd)/testbin
-test: generate fmt vet manifests
-	mkdir -p ${ENVTEST_ASSETS_DIR}
-	test -f ${ENVTEST_ASSETS_DIR}/setup-envtest.sh || curl -sSLo ${ENVTEST_ASSETS_DIR}/setup-envtest.sh https://raw.githubusercontent.com/kubernetes-sigs/controller-runtime/{{ .ControllerRuntimeVersion }}/hack/setup-envtest.sh
-	source ${ENVTEST_ASSETS_DIR}/setup-envtest.sh; fetch_envtest_tools $(ENVTEST_ASSETS_DIR); setup_envtest_env $(ENVTEST_ASSETS_DIR); go test ./... -coverprofile cover.out
+test: generate fmt vet manifests tooling 
+	$(TOOLING); go test ./... -coverprofile cover.out
+
+# get the kubebuilder tooling
+tooling:
+ifeq (, $(shell which tooling))
+	@{ \
+	set -e ;\
+	TOOLING_TMP_DIR=$$(mktemp -d) ;\
+	cd $$TOOLING_TMP_DIR ;\
+	go mod init tmp ;\
+	go get sigs.k8s.io/kubebuilder/tooling ;\
```

Comment:
why not just scaffold out `/.tooling` with a go.mod,
then this becomes `cd $PROJECT/.tooling; go run sigs.k8s.io/kubebuilder/tooling` (FWIW, I'm not sure we need do the whole "is there a predefined helper binary" dance here -- we can just build the right nobs into the helper binary)

Reply:
Done.